Source/Shaders/ModelExperimental/FeatureStageVS.glsl

```diff
@@ -0,0 +1,13 @@
+void featureStage()
+{
+    float featureId = FEATURE_ID_ATTRIBUTE;
+    if (featureId < model_featuresLength)
+    {
+        model_featureId = featureId;
+        model_featureSt = computeSt(featureId);
+    }
+    else
+    {
+        model_featureId = model_featuresLength + 1.0;
+    }
```

Comment:
Maybe add a code comment to the top that says this is only called for feature id attributes.